data/kubemacpool/003-deployment.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
   labels:
     control-plane: mac-controller-manager
     controller-tools.k8s.io: "1.0"
+      app: kubemacpool
```

Comment:
invalid indentation

Reply:
This label with find the deployment. I think you also want to add the label here :

https://github.com/kubevirt/cluster-network-addons-operator/pull/243/files#diff-bd563cbfdbacfe02cc6d35d6518a2d1eR21

Line 21

Reply:
Done